src/ui/selection_plugin.h

```diff
@@ -12,6 +12,7 @@ struct State;
 class Selection_Menu : public FishUIViewerPlugin {
 public:
     Selection_Menu(State& state);
+    void get_window_size(GLFWwindow* handle, int* width, int* height);
```

Comment:
It looks like there is more than one version of `get_window_size`. This one, and the one in `state`. Let's delete this one.